integration/configs/media_capture_test.config.json

```diff
@@ -0,0 +1,26 @@
+{
+    "`": 1,
```

Comment:
When do we use this config :D

Reply:
Currently it's not being used in the integration tests. I plan to add this to the integration test workflow for every PR.